blog/why-theres-no-new-episode-this-week.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Why There's No New Episode This Week
+publish: 2020-06-01T10:19:30
+slug: why-theres-no-new-episode-this-week
+---
+
+Due to current events happening in the United States regarding George Floyd and the protests to bring about changes to systemic racial injustice and police brutality, we will not be releasing a new episode this week and will be donating our sponsorship profits to [Campaign Zero](https://www.joincampaignzero.org/) from last week's episode.
+
+If you would also like to lend your support, here are a couple ways you can do so.
+
+1. Donate
+
+- [Campaign Zero](https://www.joincampaignzero.org/)
+- [NAACP Legal Defense Fund](https://www.naacpldf.org/)
+- [Minnesota Freedom Fund](https://minnesotafreedomfund.org/)
+- [Black Visions Collective](https://secure.everyaction.com/4omQDAR0oUiUagTu0EG-Ig2)
+- [Reclaim the Block](https://secure.everyaction.com/zae4prEeKESHBy0MKXTIcQ2)
+
+2. Read about it
+
+- [How To Be Antiracist, Ibram X Kendi](https://bookshop.org/books/how-to-be-an-antiracist/9780525509288)
+- [So You Want To Talk About Race, Ijeoma Oluo](https://bookshop.org/books/so-you-want-to-talk-about-race/9781580058827)
+
+Thanks for your understanding and we hope you all stay safe amidst these trying times.
```

Comment:
Thank you for understanding; we hope you all are all safe and well? Sounds a bit like a run on without at least a comma or something 🤔

Reply:
```suggestion
Thanks for your understanding. We hope you all stay safe amidst these trying times.
```

What do you think @tessaSac?